app.js

```diff
@@ -22,6 +23,16 @@ app.use('/scripts', express.static(path.join(__dirname, 'scripts')));
 app.use('/stylesheets', express.static(path.join(__dirname, 'stylesheets')));
 app.use('/lib', express.static(path.join(__dirname, 'lib')));
 
+//Handeling GET and POST request in Express
+app.post('/addUser',(request,response) => {
```

Comment:
in short, it is more idiomatic to have `/users`:

Please see https://google.aip.dev/133 and we can discuss further if you have any questions.

Reply:
In simple terms - you have a resource "User". You want to either create, update, delete, get.

You can map specific http verb against these actions and register your routes accordingly.

https://www.restapitutorial.com/lessons/httpmethods.html#:~:text=The%20primary%20or%20most%2Dcommonly,but%20are%20utilized%20less%20frequently.